x/perpetual/module.go

```diff
@@ -144,7 +144,7 @@ func (am AppModule) ExportGenesis(ctx sdk.Context, cdc codec.JSONCodec) json.Raw
 }
 
 // ConsensusVersion is a sequence number for state-breaking change of the module. It should be incremented on each consensus-breaking change introduced by the module. To avoid wrong/empty versions, the initial version should be set to 1
-func (AppModule) ConsensusVersion() uint64 { return 10 }
+func (AppModule) ConsensusVersion() uint64 { return 11 }
```

Comment:
@cryptokage1996 the module version was already incremented for the next upgrade please stick with v10 and update the v10 migrator function for any additional data structure changes otherwise the software upgrade will fail